crates/ironclaw_gateway/static/styles/surfaces/jobs.css

```
@@ -768,14 +907,9 @@
   width: 6px;
   height: 6px;
-  border-radius: 50%;
-  background: var(--accent);
-  animation: statusPulse 2s ease-out infinite;
-}
-
-@keyframes statusPulse {
-  0% { transform: translateY(-50%) scale(0.8); opacity: 0.6; }
-  100% { transform: translateY(-50%) scale(1.8); opacity: 0; }
+  border-radius: var(--v2-radius-full);
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)